packages/odyssey/src/scss/components/_toast-pen.scss

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.ods-toast-pen {
+  display: grid;
+  position: fixed;
+  z-index: 1;
```

Comment:
I've been fiddling with this a bit. For now, I think it's best if product teams escalate their `z-index` if it becomes a problem. Your point about Modal is well taken though - the system needs an internal answer for this.

You comfortable with this as-is? I can create a followup ticket for @arnoldsandoval-okta and I to noodle on how to handle z-index without setting up a race.

Reply:
This z-index callout seems like a broader question, one that we should partner with the WC team on. This specific toast pen issue wouldn't be an issue if you could build a product end-to-end with Odyssey. However, Odyssey isn't that kind of system right now - and it will likely never be. In other words, there will always be a need for some amount of custom components within a product.

Reply:
I feel like I will be more comfortable if this was above the z-index level of the modal bg. But okay, if you want to do that for later.

Reply:
@santhoshbalakrishnan I've opened this and flagged it as high priority (patch release candidate) in ZenHub: https://github.com/okta/odyssey/issues/703 

I think we'd be safe just bumping up the `z-index` here, but I want to ensure our solution is intentional and will grow with ODS. We should also supply some guidance + try to ensure we aren't updating `z-index` every time we add something `absolute` or `fixed`.